Actions/IncrementVersionNumber/IncrementVersionNumber.ps1

```diff
@@ -19,92 +19,50 @@ $telemetryScope = $null
 
 try {
     . (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-Helper.ps1" -Resolve)
+    Import-Module (Join-Path -path $PSScriptRoot -ChildPath "IncrementVersionNumber.psm1" -Resolve)
+
     $serverUrl, $branch = CloneIntoNewFolder -actor $actor -token $token -updateBranch $updateBranch -DirectCommit $directCommit -newBranchPrefix 'increment-version-number'
     $baseFolder = (Get-Location).path
     DownloadAndImportBcContainerHelper -baseFolder $baseFolder
 
-    import-module (Join-Path -path $PSScriptRoot -ChildPath "..\TelemetryHelper.psm1" -Resolve)
+    Import-Module (Join-Path -path $PSScriptRoot -ChildPath "..\TelemetryHelper.psm1" -Resolve)
     $telemetryScope = CreateScope -eventId 'DO0076' -parentTelemetryScopeJson $parentTelemetryScopeJson
 
+    # Validate version number. It should match <Major>.<Minor> or +<Major>.<Minor>
+    if(-not ($versionNumber -match '^\+?\d+\.\d+$')) {
```

Comment:
Could we support build and revision number as well? So e.g. 8.0.1 would also be a valid version

Reply:
I think AL-Go sets the build number at some point.

The PR doesn't aim to support more scenarios. Just fix the issue and refactor to allow testing.